app/views/configuration/show.html.haml

```diff
@@ -1 +1,2 @@
-#main_div= render :partial => @tabform
+- if %w(ui_1 ui_2 ui_3 ui_4 ui_5).include?(@tabform)
+  #main_div= render :partial => @tabform
```

Comment:
Looks like Brakeman is not liking the ivar that it traces back to an input param.  May have to do a case statement with each when rendering a string, but probably better to move this whole thing to Ruby, instead of HAML.  Up to you.